apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/StargateBridgeInterceptor.java

```diff
@@ -0,0 +1,34 @@
+package io.stargate.sgv2.api.common.grpc;
+
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.Metadata;
+import io.grpc.MethodDescriptor;
+import io.quarkus.grpc.GlobalInterceptor;
+import io.stargate.sgv2.api.common.StargateRequestInfo;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+@GlobalInterceptor
+@ApplicationScoped
+public class StargateBridgeInterceptor implements ClientInterceptor {
+  @Inject StargateRequestInfo requestInfo;
```

Comment:
Currently, this is causing tests to fail because it's not allowed...

An exception is thrown saying "No Resteasy request in progress", but it's really hard to figure out why; any ideas? Maybe using GlobalInterceptor somehow makes this interceptor not occur during a request?

Reply:
sure `@StargateRequestInfo` has the request scope, but no test should fail here.. Which one is failing?

Reply:
BridgeDataStorePropertiesConfigurationTest.dataStoreFromBridge

I made the changes you requested below, but there is a failure in that tests because injection of StargateRequestInfo is failing...

Reply:
Yea, good catch then.. We need to find a way to check if the `StargateRequestInfo` is available in the interceptor.. This means you need to inject the `Instance<StargateRequestInfo>` and then check if the instance is available..